oak_functions_containers_launcher/src/lib.rs

```diff
@@ -32,8 +32,8 @@ use tokio::time::Duration;
 use tonic::transport::Endpoint;
 
 pub struct UntrustedApp {
-    launcher: Launcher,
     oak_functions_client: GrpcOakFunctionsClient<tonic::transport::channel::Channel>,
+    launcher: Launcher,
```

Comment:
Is this really semantically relevant? I would never expected it :)
If there is no better way of doing it, could you add a comment? And would it be possible to add a simple test that would catch a regression in case the fields are swapped again?

Reply:
It's semantically relevant as Rust drops fields in declaration order, but here it doesn't matter that much any more because of the other changes.